core/network.py

```diff
@@ -124,15 +130,17 @@ def new_network_events(configuration):
                         ip_src = byte_to_str(line[1])
                         port_dest = int(line[2])
                         port_src = int(line[3])
-                        if (netaddr.valid_ipv4(ip_dest) or netaddr.valid_ipv6(ip_dest)) \
-                                and ip_dest not in ignore_ip_addresses \
-                                and ip_src not in ignore_ip_addresses \
-                                and port_dest not in ignore_ports \
-                                and port_src not in ignore_ports:
-                            # ignored ip addresses and ports in python - fix later
+                        if netaddr.valid_ipv4(ip_dest) or \
+                           netaddr.valid_ipv6(ip_dest) \
```

Comment:
I see you removed the brackets around ( netaddr.valid_ipv4(ip_dest) or  netaddr.valid_ipv6(ip_dest) ) . Would the behaviour remain the same since this entire statement is supposed to be one condition?

Reply:
my bad! Thanks for pointing that out